homeassistant/components/ondilo_ico/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "domain": "ondilo_ico",
+  "name": "Ondilo ICO",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/ondilo_ico",
+  "requirements": [
+    "ondilo==0.2.0"
+  ],
+  "ssdp": [],
```

Comment:
Please remove empty items.

Reply:
Done in PR https://github.com/home-assistant/core/pull/44837